api_fhir_r4/serializers/communicationSerializer.py

```diff
@@ -20,4 +20,9 @@ def create(self, validated_data):
         from core import datetime
         copied_data['feedback_date'] = datetime.datetime.now()
         obj = Feedback.objects.create(**copied_data)
+        imis_claim = Claim.objects.get(id=claim, validity_to__isnull=True)
```

Comment:
Why is there a `validity_to__isnull=True`?

Reply:
it's often used in module to filter and get an object. That being said, I'll remove it as in the module Claim they get Claim with id without it.